### PR TITLE
Add 2019-05-17-all-hands-meeting notes

### PR DESCRIPTION
**Pull request by garrying**

### CODE REVIEW

File: 2019-05-15-all-hands-meeting.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+[:date: Calendar][cal] | [:cat: GitHub][gh] | [:open_file_folder: GDrive][gdrive]
+
+Time: 17:00 - 18:00 EDT  
+Location: https://meet.jit.si/offline  
+Attending: 
+- yurko
+- elon
+- garry
+- dc (cultivator)
+- ben
```
Comment (patcon) [inline 2019-05-15-all-hands-meeting.md]:
```suggestion
- ben
- patcon
```

File: 2019-05-15-all-hands-meeting.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+Notetaker:  
+- [ ] benedict  
+- [~] dc  
+- [~] elon  
+- [~] garry
```
Comment (patcon) [inline 2019-05-15-all-hands-meeting.md]: Ok, I've gotta ask -- what does this mean? Is it, people who've done it already?

Comment (dcwalk) [reply 2019-05-15-all-hands-meeting.md]: yeah, because we immediately got out of sync it was a way to see who had taken notes

**Comment by dcwalk** [comment]
Thanks!

**Pull request merged**